### DISCUSSION TITLE
dns - prefetching

### DISCUSSION THREAD

**Discussion by llightcb** [DNSCrypt/dnscrypt-proxy#2451] [Q&A] [answered]
concerned. again. 

should [dns -  prefetching](https://www.usenix.org/legacy/event/leet10/tech/full_papers/Krishnan.pdf)   (pdf) be disabled in different browsers, or is this paper just information from an aluminium hut point of view, resp. outdated? unfortunately, this question is only loosely related to dnscrypt-proxy, so sry.

( + [this](https://www.chromium.org/developers/design-documents/dns-prefetching/) )

**Comment by o101010** [comment, marked as answer]
Hi,

As the prefetching are done by your web browser, I don't see how DNS resolvers or DNS proxies can block it. They basically reply to DNS queries, so they are not able to figure out if the request google.com (for example) come from the user or from browser prefetching.

**Reply by llightcb** [reply]
i was overconcerned when i wrote the post.

thx.
